health/micro-ui/web/micro-ui-internals/packages/modules/microplan/README.md

```
@@ -0,0 +1,96 @@
+## Documentation
+
+Documentation Site (https://core.digit.org/guides/developer-guide/ui-developer-guide/digit-ui)
+
+[Microplan Module Documentation](https://docs.digit.org/public-health/microplanning/setup/configuration)
+
+## Maintainer
+
+- [nipunarora-egov](https://www.github.com/nipunarora-egov)
+
+
+### Published from DIGIT Frontend 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)